src/pages/video-page/VideoPage.jsx

```diff
@@ -0,0 +1,107 @@
+import { useDocTitle } from "../../hooks/useDocTitle";
+import axios from "axios";
+import "./videopage.css";
+import { useLocation, useParams } from "react-router-dom";
+import { Card, Loader, PlaylistModal, SmashPlayer } from "../../components";
+import { videos } from "../../data/video-data";
+import { formatDate } from "../../hooks/formatDate";
+import {
+  AddToLikesIcon,
+  AddWatchLaterIcon,
+  PlaylistsAddNewIcon,
+} from "../../components/header/icons";
+import { useState, useEffect } from "react";
+
+export function VideoPage() {
+  useDocTitle("Video Page - SmashTube - Manoj Sarna");
+  const { videoId } = useParams();
+  const [loading, setLoading] = useState(true);
+  const [videoDetails, setVideoDetails] = useState();
+  const [show, setShow] = useState(false);
```

Comment:
suggestion: write specific names. Here `show` and `setShow` does not imply `modal` by default. Maybe use `showModal` and `setShowModal`